src/network/network_client.cpp

```diff
@@ -1120,6 +1120,7 @@ NetworkRecvStatus ClientNetworkGameSocketHandler::Receive_SERVER_COMPANY_UPDATE(
 {
 	if (this->status < STATUS_ACTIVE) return NETWORK_RECV_STATUS_MALFORMED_PACKET;
 
+	static_assert(sizeof(_network_company_passworded) <= sizeof(uint16));
```

Comment:
Using `<=` feels weird when the next line assign a 16bits value to it.

Reply:
It's more to check that it fits within the network packet, rather checking the packet is exactly the right length. I guess it could just be == though.